docs/2.8.0/docs/getting-started/manual-download.rst

```diff
@@ -27,7 +35,7 @@ To do that:
    Alternatively, if keyservers do not work for you (we are having a bit of
    trouble getting them to work reliably for us), you can find the full public
    key at the bottom of this page.
-4. Once the key is imported, you can ask ``gpg`` to verify that the file you have downloaded has indeed been signed by that key. Continuing with our example of 2.0.0 on macOS, you should have both files in the current directory and run::
+5. Once the key is imported, you can ask ``gpg`` to verify that the file you have downloaded has indeed been signed by that key. Continuing with our example of 2.0.0 on macOS, you should have both files in the current directory and run::
```

Comment:
Here as well, you should replace `2.0.0` with `2.8.0` everywhere for consistency.

That is in this sentence: `Continuing with our example of 2.0.0 on macOS, ...`